### PR TITLE
Content migration

### PR DESCRIPTION
**Pull request by vinicios-gomes**

### CODE REVIEW

**Review by vitormattos — CHANGES_REQUESTED** [review]
The content was not moved, all was deleted and new files created. To keep the history, a file should have been renamed and the contents changed to fit the new layout requirements.

For example:
The new site has the Patreon logo, the best way to preserv history thing would be to move the `source/assets/images/support/patreon.png` file to the new path and not delete it and add it again.

File: src/content/about-us/nossos-cooperados.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Nossos Cooperados"
```
Comment (vitormattos) [inline src/content/about-us/nossos-cooperados.md:2]: This is a list, I think will be more easy iterate on an json o yaml list of objects to generate the HTML.

File: src/content/apoie.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Apie"
```
Comment (vitormattos) [inline src/content/apoie.md:2]:
```suggestion
title: "Apoie"
```

File: src/content/home.md [hunk 3]
```diff
@@ -1,4 +1,85 @@
 ---
 title: "Home Page"
```
Comment (vitormattos) [inline src/content/home.md:2]: The content of this page sounds like a list, I believe will be best implement json or yaml list to generate this page

File: src/content/products-and-services/services.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
+---
+title: "Serviços"
```
Comment (vitormattos) [inline src/content/products-and-services/services.md:2]: This also sounds a list of items

**Review by vitormattos — CHANGES_REQUESTED** [review]
favicon was removed?

File: src/content/about-us/nossos-cooperados.md (hunk 1 above)
Comment (vinicios-gomes) [reply src/content/about-us/nossos-cooperados.md:2]: It's not a list, it's the header of the file.
For more information: https://jekyllrb.com/docs/front-matter/

Comment (vinicios-gomes) [reply src/content/about-us/nossos-cooperados.md:2]: It's not a list, it's the header of the file.
For more information: https://jekyllrb.com/docs/front-matter/

File: src/content/home.md (hunk 3 above)
Comment (vinicios-gomes) [reply src/content/home.md:2]: It's not a list, it's the header of the file.
For more information: https://jekyllrb.com/docs/front-matter/

File: src/content/products-and-services/services.md (hunk 4 above)
Comment (vinicios-gomes) [reply src/content/products-and-services/services.md:2]: It's not a list, it's the header of the file.
For more information: https://www.npmjs.com/package/front-matter

File: src/content/about-us/nossos-cooperados.md (hunk 1 above)
Comment (vitormattos) [reply src/content/about-us/nossos-cooperados.md:2]: Is impossible mark all the document and I put the comment on first line.

Comment (vinicios-gomes) [reply src/content/about-us/nossos-cooperados.md]: Thanks for the improvement, you're right, lists and json better work on this case.

File: src/content/home.md (hunk 3 above)
Comment (vinicios-gomes) [reply src/content/home.md]: Thanks for the improvement, you're right, lists and json better work on this case.

File: src/content/products-and-services/services.md (hunk 4 above)
Comment (vinicios-gomes) [reply src/content/products-and-services/services.md]: Thanks for the improvement, you're right, lists and json better work on this case.

**Review by liviacarolgouvea — APPROVED** [review]

**Review by DaianeAlvesRJ — APPROVED** [review]

**Review by vitormattos — CHANGES_REQUESTED** [review]
I believe that `source/index.blade.php` is equivalent of `src/content/home.json` and will be good use rename (or move) instead of delete and add

**Review by AntonioCoelhoDev — APPROVED** [review]
Cool

**Pull request merged**